js/index.js

```diff
@@ -1,83 +1,93 @@
-
-const today = new Date ();
+const today = new Date();
 let thisYear = today.getFullYear();
 
 ///SKILLS///
 
-let skills = ["HTML", "CSS", "JavaScript", "GitHub"]
+let skills = ["HTML", "CSS", "JavaScript", "GitHub"];
 
 const skillsSection = document.querySelector("#skills");
 
-const skillsList = skillsSection?.querySelector('ul');
+const skillsList = skillsSection?.querySelector("ul");
 
-for (let i = 0; i<skills.length; i++) {
-      const skill = document.createElement('li');
-      skill.innerHTML =`${skills[i]}`;
-      skillsSection?.appendChild(skill);
+for (let i = 0; i < skills.length; i++) {
+  const skill = document.createElement("li");
+  skill.innerHTML = `${skills[i]}`;
+  skillsSection?.appendChild(skill);
 }
 
-const technicalSkills = document.createElement('h3');
-      technicalSkills.className = 'technical-skills';
-      technicalSkills.innerHTML = "Technical";
+const technicalSkills = document.createElement("h3");
+technicalSkills.className = "technical-skills";
+technicalSkills.innerHTML = "Technical";
 
 const skillsHeader = document.querySelector("#skills-header");
 
 skillsHeader?.appendChild(technicalSkills);
 
+///PROJECTS///
+
+const projectSection = document.querySelector("#projects");
+const projectList = projectSection?.querySelector("ul");
+
+fetch(`https://api.github.com/users/AmandaHockmuth/repos`)
+  .then((response) => {
+    if (!response.ok) {
+      throw new Error(`${response.status}`);
+    }
+    return response.json();
+  })
+  .then((repositories) => {
+    for (i = 0; i < repositories.length; i++) {
```

Comment:
This works fine. But it would be a lot easier to just use forEach instead of a for loop: 
  .then((repositories) => {
    repositories.forEach((repo) => {
      const project = document.createElement("li");
      project.innerHTML = `${repo.name}`;
      projectList.appendChild(project);
    });
  })